src/theory/quantifiers/sygus/sygus_enumerator.cpp

```diff
@@ -31,10 +31,71 @@ SygusEnumerator::SygusEnumerator(TermDbSygus* tds, SynthConjecture* p)
 
 void SygusEnumerator::initialize(Node e)
 {
+  Trace("sygus-enum") << "SygusEnumerator::initialize " << e << std::endl;
   d_enum = e;
   d_etype = d_enum.getType();
+  Assert(d_etype.isDatatype());
+  Assert(d_etype.getDatatype().isSygus());
   d_tlEnum = getMasterEnumForType(d_etype);
   d_abortSize = options::sygusAbortSize();
+
+  // Get the statically registered symmetry breaking clauses for e, see if they
+  // can be used for speeding up the enumeration.
+  NodeManager* nm = NodeManager::currentNM();
+  std::vector<Node> sbl;
+  d_tds->getSymBreakLemmas(e, sbl);
+  Node ag = d_tds->getActiveGuardForEnumerator(e);
+  Node truen = nm->mkConst(true);
+  // use TNode for substitute below
+  TNode agt = ag;
+  TNode truent = truen;
+  Assert(d_tcache.find(d_etype) != d_tcache.end());
+  const Datatype& dt = d_etype.getDatatype();
+  for (const Node& lem : sbl)
+  {
+    if (!d_tds->isSymBreakLemmaTemplate(lem))
+    {
+      // substitute its active guard by true and rewrite
+      Node slem = lem.substitute(agt, truent);
+      slem = Rewriter::rewrite(slem);
+      // break into conjuncts
```

Comment:
what if the static sym break lemma is a disjunction of lemmas?

Reply:
Then we cant use it :)

Reply:
OK. Add a comment and maybe open an issue.

Reply:
I dont think this is issue worthy (it's not immediately clear that we should use them).  But I'll comment that we dont use them currently.